### ISSUE TITLE
Installing a specific version of the plugin results in a loading error

### ISSUE THREAD

**Issue by mildebrandt** [turbot/steampipe-plugin-aws#697]
When I install a specific version, I get the following error after running `steampipe query`:
```
$ steampipe plugin install aws@0.36.0

Installed plugin: aws@0.36.0 v0.36.0
Documentation:    https://hub.steampipe.io/plugins/turbot/aws

$ steampipe query
Welcome to Steampipe v0.9.0
For more information, type .help
> 
Error: failed to load connection 'aws': no plugin installed matching aws
/Users/chrism/.steampipe/config/aws.spc:1,1-11
```

I'm able to edit the `~/.steampipe/config/aws.spc` file and change the plugin to `aws@0.36.0` to get things working again.

**Steampipe version (`steampipe -v`)**
v0.9.0

**Plugin version (`steampipe plugin list`)**
v0.36.0

**Comment by rajlearner17** [comment]
@mildebrandt hope you doing well. The issue is waiting long, thanks for waiting. We are looking into this, will provide an update when ready.

**Comment by mildebrandt** [comment]
Commenting to remove the stale label.

**Comment by mildebrandt** [comment]
That did not remove the stale label. :(

**Comment by cbruno10** [comment]
@mildebrandt I've gone ahead and removed the stale label, I'm looking into the status of https://github.com/turbot/steampipe/issues/1052 as a potential resolution to this issue, and will post updates in that issue and/or this one to provide more updates.

**Comment by mildebrandt** [comment]
Thanks!

**Comment by mildebrandt** [comment]
I'm hoping to see this get fixed.

**Comment by cbruno10** [comment]
Hey @mildebrandt , I believe this issue should be resolved in Steampipe v0.13.1, as per https://github.com/turbot/steampipe/issues/1675 (which replaced https://github.com/turbot/steampipe/issues/1052).

If you encounter this bug again, please comment and re-open so we can have another look. Thanks!

**Comment by mildebrandt** [comment]
Verified on my side as well. Sorry, I should have closed this when I saw the other issue get resolved. Thanks for following up!
